kriswallsmith/buzz/0.17/config/packages/buzz.yaml

```diff
@@ -0,0 +1,35 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
+    _defaults:
+        public: false
+
+    # Tell the auto wire system to use "FuleGetContents" client as default client.
+    Buzz\Client\BuzzClientInterface: '@Buzz\Client\FileGetContents'
```

Comment:
the previous version of the PR was using Curl as default implementation. Was it expected to change ?

Reply:
Good question. If I do `new Browser()` I will get `FileGetContents` as client. I thought the service definition should be the same. 

I think we use `FileGetContents` because that one "will always work". Curl has a dependency on a PHP extension.